test/unit/services/ipfetcher/fetch-ip-info.spec.ts

```diff
@@ -59,6 +60,28 @@ describe("IpFetcher - fetchIPInfo", () => {
   })
 })
 
+describe("ipaddr.js", () => {
```

Comment:
this needs to be moved to test domain folder and description should be isPrivateIp (we are testing domain function here)